Client/Frontend/Widgets/HistoryBackButton.swift

```diff
@@ -40,7 +40,11 @@ class HistoryBackButton: UIButton {
         backgroundColor = UIColor.white
 
         chevron.snp.makeConstraints { make in
-            make.leading.equalTo(self).offset(HistoryBackButtonUX.HistoryHistoryBackButtonHeaderChevronInset)
+            if #available(iOS 11.0, *) {
```

Comment:
If you could please update this PR to also use `view.safeArea` that would be awesome!